src/lib/common/database/database.ts

```diff
@@ -0,0 +1,21 @@
+import * as common from './methods/common';
```

Comment:
We are duplicating the code in `common` and `utils`. If the idea is having two repositories for the online service, we should thing in a solution to not duplicate this code.

Thoughts?

Reply:
The idea is to have just one repo once we migrate everything to serverless

Reply:
Yes, a lot of common parts are duplicated, I didn't want to work on that refactor in this PR. I think it's best to let it be duplicated, test out the changes, and on a separate PR do the refactor to dedupe?